libxaw.yaml

```diff
@@ -1,7 +1,6 @@
-# Generated from https://git.alpinelinux.org/aports/plain/main/libxaw/APKBUILD
 package:
   name: libxaw
-  version: 1.0.14
+  version: 1.0.15
```

Comment:
this is being updated in this PR because https://www.x.org have stopped producing file extensions `bz2`, so the next auto update would fail unless updated.